src/GitVersion.Core.Tests/VersionCalculation/BaseVersionCalculatorTests.cs

```diff
@@ -26,10 +26,10 @@ public void ChoosesHighestVersionReturnedFromStrategies()
 
         var baseVersion = versionCalculator.GetBaseVersion();
```

Comment:
I don't mind using tuples if we consistently use the tuple syntax introduced in C# 7, so we avoid having non-descriptive properties like `Item1` in the codebase.

```suggestion
        var (baseVersion, _) = versionCalculator.GetBaseVersion();
```

Reply:
Jep good point. I'm going to change it. Actually it's not so ideal to use Tuples as a result for GetBaseVersion. But here I used is because it will be fused with NextVersionCaluclator in the v6 refactoring.